.circleci/config.yml

```diff
@@ -107,16 +107,28 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - checkout_code
+      - checkout_code:
+          filters:
+            tags:
+              only: /v.*/
       - install-dependencies:
           requires:
             - checkout_code
+          filters:
+            tags:
+              only: /v.*/
       - test:
           requires:
             - install-dependencies
+          filters:
+            tags:
+              only: /v.*/
       - build:
           requires:
             - install-dependencies
+          filters:
+            tags:
```

Comment:
unfortunately we have to add `tags` to all jobs, as stated in the documentation:
> a job must have a filters tags section to run as a part of a tag push and all its transitively dependent jobs must also have a filters tags section

see also:
https://circleci.com/docs/2.0/workflows/#git-tag-job-execution